src/controllers/bulk/bulkcontroller.cpp

```diff
@@ -5,9 +5,14 @@
 #include "controllers/bulk/bulksupported.h"
 #include "controllers/defs_controllers.h"
 #include "moc_bulkcontroller.cpp"
+#include "util/cmdlineargs.h"
 #include "util/time.h"
 #include "util/trace.h"
 
+namespace {
+constexpr int kBulkSendTimeout = 5000; // In milliseconds
```

Comment:
lets leverage `<chrono>`.
```suggestion
constexpr std::seconds kBulkSendTimeout = 5s;
```